File: level1/javascript/두 개 뽑아서 더하기 68644.js
function solution(numbers) {
    let answer = new Set();
    for(let i = 0; i < numbers.length - 1; i++)
        for(let j = i+1; j < numbers.length; j++)
            answer.add(numbers[i] + numbers[j]);

    return [...answer].sort((a,b)=>a-b); // Set -> 배열로 변환, 오름 차순 정렬
} 

File: level1/kotlin/두 개 뽑아서 더하기 68644.kt
class Solution {
    fun solution(numbers: IntArray): IntArray {
        val answer = mutableSetOf<Int>()
        for (i in 0..numbers.size - 2) {
            for (j in i + 1 until numbers.size) 
                answer.add(numbers[i] + numbers[j])
        }
        return answer.toIntArray().sortedArray()
    }
}